validator/client/beacon-api/beacon_api_node_client.go

```diff
@@ -18,14 +18,18 @@ import (
 
 type beaconApiNodeClient struct {
 	fallbackClient  iface.NodeClient
-	jsonRestHandler jsonRestHandler
-	genesisProvider genesisProvider
+	jsonRestHandler JsonRestHandler
+	genesisProvider GenesisProvider
 }
 
 func (c *beaconApiNodeClient) GetSyncStatus(ctx context.Context, _ *empty.Empty) (*ethpb.SyncStatus, error) {
 	syncingResponse := node.SyncStatusResponse{}
-	if _, err := c.jsonRestHandler.GetRestJsonResponse(ctx, "/eth/v1/node/syncing", &syncingResponse); err != nil {
-		return nil, errors.Wrap(err, "failed to get sync status")
+	errJson, err := c.jsonRestHandler.Get(ctx, "/eth/v1/node/syncing", &syncingResponse)
```

Comment:
As per previous comment, should we have all these endpoints here as constants as well?